recipe/activate.sh

```diff
@@ -16,7 +16,7 @@ if [ "${CONDA_BUILD:-0}" = "1" ]; then
     CUDA_LDFLAGS="${CUDA_LDFLAGS} -L${BUILD_PREFIX}/${targetsDir}/lib"
     CUDA_LDFLAGS="${CUDA_LDFLAGS} -L${BUILD_PREFIX}/${targetsDir}/lib/stubs"
     # Needed to fix cross compilation
```

Comment:
Can you add to this comment?

Reply:
Sure!

Reply:
Updated the comment. Happy to take any revisions